src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -131,6 +132,17 @@ export const DatasetSearchTable = (props) => {
         }
       });
 
+      filterTerms.push({
```

Comment:
With only two cases I didn't think it was worth refactoring, but with three maybe it's time? Something like
```
   const addTerms = (createFilters) => {
      filterTerms.push({
        'bool': {
          'should': createFilters()
        }
      });
  }

  addTerms(() => {
     filters.accessManagement.map(term => ({
                'term': {
                  'accessManagement': term
                }
              }))
  );
  addTerms(...);
  addTerms(...);
```

Reply:
Hrm, I'm not sure this is clearer, and it's possible that we might have a filter that requires a `must` or some other boolean term.